alsid-syslog-proxy/azuredeploy.parameters.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "value": "GEN-UNIQUE"
+        },
+        "vmAdminUserName": {
+            "value": "GEN-UNIQUE"
+        },
+        "vmAdminPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "workspaceId": {
+            "value": "GEN-GUID"
+        },
+        "primaryKey": {
+            "value": "GEN-UNIQUE"
+        },
+        "_artifactsLocation": {
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
this still needs to be done

Reply:
I don't understand what needs to be changed, I put this value in the parameters because the default URI for artifacts location doesn't exist yet since the files are not yet on master on this repository, Letting the default value will therefore fail the deployment since the custom script extension will not be able to find the script.
Should I ignore this problem and use the artifacts location default value anyway or am I missing something?